frontend/src/app.css

```
@@ -0,0 +1,79 @@
+  than switched per page.
+
+  - Display/utility type is monospace throughout (JetBrains Mono): headlines,
+    IDs, ports, domains, env vars all read like they came from a terminal,
+    because on this product they basically did.
+  - Body copy is IBM Plex Sans — quieter, built for paragraphs.
+  - One accent (red-600/700, carried over from the original htmx templates)
+    handles brand and interactive state. Green/amber are reserved strictly
+    for real infra state — never decorative.
+*/
+
+@import url("https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@400;500;600&display=swap");
```

Stylelint (17.14.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@400;500;600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@400;500;600&display=swap"" (import-notation)

(import-notation)